python_modules/dagster/dagster_tests/core_tests/asset_defs_tests/test_assets.py

```diff
@@ -127,6 +131,27 @@ def ma_op():
     assert subset.group_names_by_key[AssetKey("b")] == "bar"
 
 
+def test_retain_partition_mappings():
+    @asset(
+        ins={"input_last": AssetIn(["input_last"], partition_mapping=LastPartitionMapping())},
+        partitions_def=DailyPartitionsDefinition(datetime.datetime(2022, 1, 1)),
+    )
+    def bar_(input_last):
+        pass
```

Comment:
The linter complains that `input_last` is an unused import. You could fix this by replacing `pass` with `del input_last`.